libcove/lib/common.py

```diff
@@ -382,14 +382,39 @@ def get_schema_obj(self, deref=False):
             return self.deref_schema(self.schema_str)
         return self._schema_obj
 
-    def get_pkg_schema_obj(self, deref=False):
+    def get_pkg_schema_obj(self, deref=False, use_extensions=False):
```

Comment:
I'm not seeing `use_extensions` var being used in the function, is there a need for it?

Reply:
Yes, this is called by some code that was already in libcove, but I think until now only used by lib-cove-ocds: https://github.com/OpenDataServices/lib-cove/blob/44f1d42c17c6097918c6d4922e0c6601944f5619/libcove/lib/common.py#L459

Reply:
Ah I see!